_layouts/post.html

```diff
@@ -8,8 +8,14 @@ <h1 class="post-title">{{ page.title }}</h1>
     <p class="post-meta">{{ page.date | date: "%b %-d, %Y" }}{% include author.html %}{% if page.meta %} • {{ page.meta }}{% endif %}</p>
   </header>
 
+  <span itemprop="keywords">
```

Comment:
Let’s make this a `ul` element with `aria-label="tags"` property. Each tag in a `li.post-badge`